tickets/models.py

```diff
@@ -121,17 +121,17 @@ def invitation(self):
 
 class TicketInvitation(models.Model):
     ticket = models.ForeignKey(Ticket, related_name='invitations', on_delete=models.CASCADE)  # TODO make this a OneToOneField
-    email_addr = models.EmailField(unique=True)
+    email_addr = models.EmailField()
     name = models.CharField(max_length=100)
-    token = models.CharField(max_length=12, unique=True)  # An index is automatically created since unique=True
+    token = models.CharField(max_length=24, unique=True)  # An index is automatically created since unique=True
     status = models.CharField(max_length=10, default='unclaimed')
 
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     class Manager(models.Manager):
         def create(self, **kwargs):
-            token = get_random_string(length=12)
+            token = get_random_string(length=24)
```

Comment:
I don't see the need to change this -- 12 characters is [71 bits of information](https://github.com/django/django/blob/master/django/utils/crypto.py#L54-L55).  While it's almost certainly a safe change to make, I'd like a little more convincing it's necessary.

Reply:
It's a token only used in a URL, therefore the length is largely irrelevant as no-one has to manually quote it. Therefore it should be as long as makes reasonable sense. 

I would argue that UUIDs have excellent properties in this regard, and are in that "as long as makes reasonable sense" category, and I looked at the `shortuuid` library which uses base57 to make 22 character long representations of UUIDs. 

Not wishing to add another dependency, I just changed it to 24 characters, which is ~143 bits of entropy.

In short - no-one is having to quote this value, so there is no reason for it not to be as secure as possible.

Reply:
I still don't understand why we need to change this (with 12 characters you're never going to get a collision, and nobody's going to bother trying to brute force a token) but don't really have a strong opinion.